java/java.hints/test/unit/src/org/netbeans/modules/java/hints/errors/CreateElementTest.java

```diff
@@ -441,6 +441,21 @@ public void test194625() throws Exception {
                 "CreateConstructorFix:(java.lang.String string):org.netbeans.test.java.hints.CreateConstructor"
         )));
     }
+
+    // the test should show that no fix is suggested for a method call where the return type is using the var.
+
```

Comment:
Please remove unnessary whitespace lines (446, 448, 454) and indent line 456 one level (see methods above)